freeform_json_to_yaml.py

```diff
@@ -0,0 +1,440 @@
+                        written_injuries[injury['name']].append(injury['location'])
+                    else:
+                        written_injuries[injury['name']] = [injury['location']]
+                injuries.append(injury)
+            new_c['injuries'] = injuries
+            if len(written_injuries) == 0:
+                unstructured_inj = 'Has no known injuries.'
+            else:
+                unstructured_inj = "Has"
+                pronoun = 'his' if demographics['sex'] == 'M' else 'her'
+                is_asthmatic = False
+                for inj_set in written_injuries:
+                    if len(written_injuries[inj_set]) == 1:
+                        if inj_set == 'Collapse':
+                            unstructured_inj += f" a collapsed {written_injuries[inj_set][0]},".lower()
+                        elif inj_set == "Shrapnel":
+                            unstructured_inj += f" shrapnel in {pronoun} {written_injuries[inj_set][0]},".lower()
+                        elif inj_set == "Amputation":
+                            unstructured_inj += f" an amputated {written_injuries[inj_set][0]},".lower()
+                        elif inj_set == "Asthmatic":
+                            is_asthmatic = True
+                        elif inj_set == 'Ear Bleed':
+                            unstructured_inj += f" an ear bleed,"
+                        else:
+                            unstructured_inj += f" a {inj_set} on {pronoun} {written_injuries[inj_set][0]},".lower()
+                            
+                    else:
+                        if inj_set == 'Collapse':
+                            unstructured_inj += f" a collapsed left and right chest,".lower()
+                        if inj_set == 'Asthmatic':
+                            is_asthmatic = True
+                        else:
+                            unstructured_inj += f" several {inj_set} injuries,".lower()
+                if unstructured_inj != 'Has':
+                    comma_separated = unstructured_inj.split(',')
+                    if len(comma_separated) == 2:
+                        unstructured_inj = comma_separated[0] + '.'
+                    else:
+                        unstructured_inj = ','.join(comma_separated[:-2]) + f'{", " if len(comma_separated) > 3 else " "}and' + comma_separated[-2] + '.'
+                else:
+                    unstructured_inj = ""
+            new_c['unstructured'] += f"{'Male' if demographics['sex'] == 'M' else 'Female'}, about {demographics['age']} years old, {POSE_MAP[c['animations']['pose']]}." 
+            if is_military:
+                new_c['unstructured'] += f" Wearing a {'US' if us_military else 'local'} military uniform."
+            new_c['unstructured_postassess'] += new_c['unstructured'] + " " + unstructured_inj
+            if is_asthmatic:
+                new_c['unstructured_postassess'] += " Patient is asthmatic."
```

Comment:
Hmm, if this injury is "hidden" then this shouldn't be added to the `unstructured_postassess`.  That said, I don't know if it's a problem to make it discoverable.  Maybe a question for a PI:  do they want ADMs to be able to discover that a patient is asthmatic in open world, or should it remain hidden?
* If the former, then change hidden to discoverable.
* If the latter, then remove this from `unstructured_postassess`.
* If they don't care, then change one or the other.

Reply:
@brianpippin

Reply:
Ideally we are making these open world experiments to be a 1:1 for the VR simulator.  Think of them as TA3 experiments independent of what TA1 is doing.  If a person in the VR can discover something, than the ADM should be able to.  If something isn't ever revealed to the human, it probably shouldn't be revealed to the ADM.  If it is never going to be revealed than I would question why we would even need it in either.  Hopefully that makes sense?  Happy to discuss on Teams or a call.

Reply:
Asthma is revealed in the sim through noisy breathing and can be answered directly if the participant asks the patient. Therefore, I would say we keep it in discoverable.